src/Plugin.php

```diff
@@ -75,12 +75,17 @@ public static function wp_enqueue_scripts() {
    * @implements post_gallery
    */
   public static function post_gallery($output = '', $atts) {
+    if (empty($atts)) {
```

Comment:
Please read [this section](http://php.net/manual/en/functions.arguments.php#functions.arguments.default) about function default arguments…

You should use `$atts = []` in the functions parameter list, than checking for empty.

Reply:
@fabianmarz I know about default arguments, sure 😄 . The problem is that the argument `$atts` is not undefined but set as `` (empty string), so the default value would not be applied. That's why I am checking for empty and then setting $args as an empty array if necessary.

Reply:
If you prefer, I can do:
```php
public static function post_gallery($output = '', $atts = []) {
    if (!is_array($atts)) {
      $atts = [];
    }
...
```
This way not having a default value for the last argument in the call (phpcs complains) is also fixed.